target/linux/at91/image/sama5.mk

```diff
@@ -19,13 +19,13 @@ define Build/at91-sdcard
 	$(BIN_DIR)/u-boot-$(DEVICE_DTS:at91-%=%)_mmc/u-boot.bin \
 	::u-boot.bin
 
-      $(if $(findstring sama5d4-xplained,$@), \
-          mcopy -i $@.boot \
+  $(if $(findstring sama5d4-xplained,$@), \
+	  mcopy -i $@.boot \
               $(BIN_DIR)/at91bootstrap-$(DEVICE_DTS:at91-%=%)sd_uboot_secure/at91bootstrap.bin \
               ::BOOT.bin,
           mcopy -i $@.boot \
               $(BIN_DIR)/at91bootstrap-$(DEVICE_DTS:at91-%=%)sd_uboot/at91bootstrap.bin \
-              ::BOOT.bin))
+              ::BOOT.bin)
```

Comment:
This looks like a fix for a bug introduced in the previous commit, please move it to the previous commit.